### PR TITLE
build setup: added new gradle task

### PR DESCRIPTION
**Pull request by arimer**
### Gradle build
-added setup task to gradle
-commited projectlibraries.overrides.properties files

### CODE REVIEW

File: projectlibraries.overrides.properties.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# To override values defined in this file create projectlibraries.overrides.properties and put overridden values there.
+mbeddr.plattform=${mbeddr.github.core.home}/code/languages
+mbeddr.sl-all=${mbeddr.github.core.home}/code/plugins
+iets3.os=${iets3.github.opensource.home}/code
```
Comment (arimer) [inline projectlibraries.overrides.properties.example]: Lines 3 and 4 does not make any sense right? @sergej-koscejev

Comment (sergej-koscejev) [reply projectlibraries.overrides.properties.example]: Why not?

File: build.gradle [hunk 2]
```diff
@@ -1,3 +1,15 @@
+import de.itemis.mps.gradle.GenerateLibrariesXml;
+
+//will pull the groovy classes/types from nexus to the classpath
+buildscript {
+    repositories {
+        maven { url 'https://projects.itemis.de/nexus/content/repositories/mbeddr' }
+    }
+    dependencies {
+        classpath 'de.itemis.mps:mps-gradle-plugin:1.0+'
```
Comment (sergej-koscejev) [inline build.gradle]: Perhaps we should fix the used plugin version to make builds reproducible? Why do we use wildcards (`1.0+`) here?

File: projectlibraries.overrides.properties.example [hunk 3]
```diff
@@ -1,5 +1,3 @@
 # To override values defined in this file create projectlibraries.overrides.properties and put overridden values there.
 mbeddr.plattform=${mbeddr.github.core.home}/code/languages
-mbeddr.sl-all=${mbeddr.github.core.home}/code/plugins
-iets3.os=${iets3.github.opensource.home}/code
```
Comment (sergej-koscejev) [inline projectlibraries.overrides.properties.example]: Okay I get what you meant now :)

**Pull request merged**